src/Ray.php

```diff
@@ -109,7 +109,8 @@ class Ray
 
     public static function create(?Client $client = null, ?string $uuid = null): self
     {
-        $settings = SettingsFactory::createFromConfigFile();
+        $configDir = (isset($_SERVER['RAY_CONFIG_DIR'])) ? $_SERVER['RAY_CONFIG_DIR'] : null;
```

Comment:
I think using `getenv()` here would be a better choice instead of directly accessing `$_SERVER`. It might also be good to do some validation of the value before passing it to `SettingsFactory`.